File: Python/car.py
from account import Account

class Car:
    id                  = int
    license             = str  
    driver              = Account 
    numberOfPassengers  = int

    def __init__(self, id, license, driver, numberOfPassengers):
        self.license = license
        self.driver = driver
        self.numberOfPassengers = numberOfPassengers
        self.id = id

File: PHP/Car.php
<?php
class Car{
    private $id;
    private $license;
    private $driver;
    private $numberOfPassengers;

    public function __construct($id, $license, $driver, $numberOfPassengers){
        $this->id = $id;
        $this->license = $license;
        $this->driver = $driver;
        $this->numberOfPassengers = $numberOfPassengers;
    }
}
?>